{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/urls.py

```diff
@@ -1,4 +1,22 @@
-from django.conf.urls import include, url
+from django.conf.urls import url, include
+{% if cookiecutter.django_registration == 'n' %}
+from .views import LoginView, LogoutView, RegistrationView, PasswordResetView, PasswordResetConfirmView
+
+authurls = [
+    url(r'^login/', LoginView.as_view(), name='user-login'),
+    url(r'^logout/', LogoutView.as_view(), name='user-logout'),
+{%- if cookiecutter.djoser != 'basic' -%}
+    url(r'^register/user/', RegistrationView.as_view(), name='user-register'),
+    url(r'^password/reset/$', PasswordResetView.as_view(), name='password-reset'),
+    url(r'^password/reset/confirm/$', PasswordResetConfirmView.as_view(), name='password-confirm'),
+{%- endif -%}
+]
+
+urlpatterns = [
+    url(r'^api/auth/', include(authurls))
```

Comment:
@jschneier wouldn't we want to the auth stuff to be at the account level? If I do it at the lower level, how do I add new url paths with `include('{{ cookiecutter.repo_name }}.account.urls')`? (assuming that's the way to grab it)